docker-compose/technical/prometheus.yml

```
@@ -0,0 +1,12 @@
+global:
+  scrape_interval: 2s
+  evaluation_interval: 2s
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['172.17.0.1:8000']
```

can it ask himself directly?
```suggestion
      - targets: ['127.0.0.1:9090']
```

---

Done